.circleci/config.yml

```diff
@@ -3,36 +3,11 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
 
-commands:
```

Comment:
looks like something went wrong on the rebase. This is all reverting changes we made since https://github.com/dwavesystems/dwave-preprocessing/pull/55 was made.

The only changes to CI should be adding 
```yml
      - run: git submodule sync
      - run: git submodule update --init
```
to most of the jobs